packages/vm/lib/state/stateManager.ts

```diff
@@ -102,8 +112,8 @@ export default class DefaultStateManager implements StateManager {
    * event. Touched accounts that are empty will be cleared
    * at the end of the tx.
    */
-  touchAccount(address: Buffer): void {
-    this._touched.add(address.toString('hex'))
+  touchAccount(address: Address): void {
```

Comment:
Name of `touchAccount` is misleading: we are touching an `Address`, not an `Account`

Reply:
well yeah, but aren't we touching the account through the address? i think the naming is fine here, but let me know if you want I can update to... `touchAddress`? It wouldn't match the rest of the stateManager interface: `getAccount`, `putAccount`, `deleteAccount`, etc.

```typescript
  getAccount(address: Address): Promise<Account>
  putAccount(address: Address, account: Account): Promise<void>
  deleteAccount(address: Address): Promise<void>
  touchAccount(address: Address): void
```

```typescript
  async _touchAccount(address: Address): Promise<void> {
    const account = await this._state.getAccount(address)
    return this._state.putAccount(address, account)
  }
```